### PR TITLE
JSUI-3087 Changed the appearance of the "no results" facet search state

### PR DESCRIPTION
**Pull request by btaillon**
https://coveord.atlassian.net/browse/JSUI-3087

Before:
image

After:
image

Inspiration (`DynamicFacet`'s search):
image

[Deploy](https://dashboard.heroku.com/pipelines/a3535101-5bbf-4a5b-a909-47fcf8c9f149)

### CODE REVIEW

**Review by ThibodeauJF — APPROVED** [review]

File: unitTests/ui/FacetSearchElementTest.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      it('searchResults contains a single element', () => {
+        expect($$(facetSearchElement.searchResults).children().length).toEqual(1);
+      });
+
+      it('renders a noResults element as a HTMLLIElement', () => {
+        expect(noResultsElement instanceof HTMLLIElement).toBeTruthy();
+      });
+
+      it('renders a noResults element with an id and a class', () => {
+        expect($$(noResultsElement).hasClass('coveo-facet-value-not-found'));
```
Comment (samisayegh) [inline unitTests/ui/FacetSearchElementTest.ts:30]: Should this expectation have a `.toBe(true)` chained to it?

**Review by samisayegh — APPROVED** [review]

File: unitTests/ui/FacetSearchElementTest.ts (hunk 1 above)
Comment (olamothe) [reply unitTests/ui/FacetSearchElementTest.ts:30]: Indeed ;)

**Review by olamothe — APPROVED** [review]

**Review by acverrette — APPROVED** [review]
All good ! Way better

File: unitTests/ui/FacetSearchElementTest.ts (hunk 1 above)
Comment (btaillon) [reply unitTests/ui/FacetSearchElementTest.ts:30]: Woops, that's right.

**Pull request merged**